### PR TITLE
feat(Storage): sample for get bucket class and location

### PR DESCRIPTION
**Pull request by yash30201**
Wrote a sample and its test for fetching and displaying the bucket's storage class and location.

### CODE REVIEW

File: storage/test/storageTest.php [hunk 1]
```diff
@@ -132,6 +131,11 @@ public function testCreateGetDeleteBuckets()
 
         $this->assertStringContainsString('Bucket Metadata:', $output);
 
+        $output = $this->runFunctionSnippet('get_bucket_class_and_location', [$bucketName]);
```
Comment (saranshdhingra) [inline storage/test/storageTest.php:134]: This needs to be called in a different test method itself.
You can use PHPunit's `@depends` annotation if you need a bucket created.

File: storage/test/storageTest.php [hunk 2]
```diff
@@ -132,6 +131,11 @@ public function testCreateGetDeleteBuckets()
...
+
+        $bucketInfo = $bucket->info();
+        $this->assertStringContainsString("Bucket: $bucketName, storage class: " . $bucketInfo['storageClass'] . ', location: ' . $bucketInfo['location'], $output);
```
Comment (saranshdhingra) [inline storage/test/storageTest.php:137]:
```suggestion
        $this->assertStringContainsString(sprintf('Bucket: %s, storage class: %s, location: %s', $bucketName, $bucketInfo['storageClass'], $bucketInfo['location']) $output);
```

File: storage/src/get_bucket_class_and_location.php [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+use Google\Cloud\Storage\StorageClient;
+
+/**
+ * Fetch the storage class and the bucket location of a given bucket.
+ *
+ * @param string $bucketName The name of your Cloud Storage bucket.
+ */
+function get_bucket_class_and_location($bucketName)
+{
+    // $bucketName = 'my-bucket';
```
Comment (saranshdhingra) [inline storage/src/get_bucket_class_and_location.php:36]: We are discussing removing this in another PR, my nit suggestion would be to remove this comment.

**Review by saranshdhingra — CHANGES_REQUESTED** [review]

File: storage/src/get_bucket_class_and_location.php [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+    $storage = new StorageClient();
+    $bucket = $storage->bucket($bucketName);
+
+    $info = $bucket->info();
+    printf('Bucket: %s, storage class: %s, location: %s' . PHP_EOL, $info['name'], $info['storageClass'], $info['location']);
```
Comment (vishwarajanand) [inline storage/src/get_bucket_class_and_location.php:42]: Break into multiple lines if exceeding 80 chars?

File: storage/src/get_bucket_class_and_location.php (hunk 3 above)
Comment (yash30201) [reply storage/src/get_bucket_class_and_location.php:36]: Removed that. Marking this as resolved.

File: storage/test/storageTest.php [hunk 5]
```diff
@@ -139,6 +139,14 @@ public function testCreateGetDeleteBuckets()
         $this->assertStringContainsString("Bucket deleted: $bucketName", $output);
     }
 
+    public function testGetBucketClassAndLocation()
+    {
+        $output = $this->runFunctionSnippet('get_bucket_class_and_location', [self::$tempBucket->name()]);
+        $bucketInfo = self::$tempBucket->info();
+
+        $this->assertStringContainsString(sprintf('Bucket: %s, storage class: %s, location: %s' . PHP_EOL, $bucketInfo['name'], $bucketInfo['storageClass'], $bucketInfo['location']), $output);
```
[the remark this answers is not in the document]
Comment (yash30201) [reply storage/test/storageTest.php:147]: Done

File: storage/src/get_bucket_class_and_location.php (hunk 4 above)
Comment (yash30201) [reply storage/src/get_bucket_class_and_location.php:42]: Apllied this everywhere required

**Review by saranshdhingra — COMMENTED** [review]
LGTM, but need to fix the CLA issue

File: storage/test/storageTest.php [hunk 6]
```diff
@@ -139,6 +138,23 @@ public function testCreateGetDeleteBuckets()
         $this->assertStringContainsString("Bucket deleted: $bucketName", $output);
     }
 
+    public function testGetBucketClassAndLocation()
+    {
+        $output = $this->runFunctionSnippet(
+          'get_bucket_class_and_location',
+          [self::$tempBucket->name()],
```
Comment (vishwarajanand) [inline storage/test/storageTest.php:145]: Convert to 4 spaces.

File: storage/test/storageTest.php [hunk 7]
```diff
@@ -139,6 +138,23 @@ public function testCreateGetDeleteBuckets()
...
+        );
+
+        $bucketInfo = self::$tempBucket->info();
+
+        $this->assertStringContainsString(sprintf(
+          'Bucket: %s, storage class: %s, location: %s' . PHP_EOL,
+          $bucketInfo['name'],
+          $bucketInfo['storageClass'],
+          $bucketInfo['location'],
```
Comment (vishwarajanand) [inline storage/test/storageTest.php:154]: Convert to 4 spaces

**Review by vishwarajanand — CHANGES_REQUESTED** [review]
LGTM
nits on formatting

**Review by vishwarajanand — APPROVED** [review]

**Pull request merged**